processor/src/chiplets/bitwise/mod.rs

```diff
@@ -249,3 +278,32 @@ pub fn assert_u32(value: Felt) -> Result<Felt, ExecutionError> {
         Ok(value)
     }
 }
+
+// BITWISE LOOKUPS
+// ================================================================================================
+#[allow(dead_code)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct BitwiseLookup {
+    op_id: Felt,
+    a: Felt,
+    b: Felt,
+    z: Felt,
```

Comment:
We could do this and (3) would be a nice benefit, if true. The reason they're `Felt` is that they're already `Felt` everywhere they come from (stack executors and Bitwise trace), so it was simpler to keep them that way than to convert to int and back. I can go either way here, so let me know if you want this change

Reply:
I'm fine with keeping as is for now. It is not clear to me whether performance will improve (as enums consume as much memory as the biggest variant, I think) - so, this would probably require benchmarking before making any changes.